packages/capabilities/bundle/config/upstream/rbac.yaml

```diff
@@ -67,30 +67,11 @@ rules:
       - get
       - list
       - watch
-  - apiGroups:
-      - run.tanzu.vmware.com
-    resources:
-      - tanzukubernetesclusters
-      - tanzukubernetesclusters/status
-    verbs:
-      - get
-      - list
-      - watch
-  - apiGroups:
-      - clusterctl.cluster.x-k8s.io
-    resources:
-      - providers
-      - providers/status
-    verbs:
-      - get
-      - list
-      - watch
   - apiGroups:
       - ""
     resources:
-      - configmaps
-      - namespaces
-      - nodes
+      - serviceaccounts
+      - secrets
```

Comment:
question: why do we need get/list secrets now ?

Reply:
the security model for capabilities allows users to provide service account name to query resources they have access to, in order to achieve that, we build a custom rest.Config for each custom resource and use that to query resources https://github.com/vmware-tanzu/tanzu-framework/blob/main/capabilities/controller/pkg/capabilities/core/capability_controller.go#L54